src/content/docs/en/guides/rss.mdx

```diff
@@ -214,6 +214,22 @@ export function GET(context) {
 }
 ```
 
+## Removing trailing slashes
+
+RSS feed has a trailing slash on the end of the links by default. Use `rss` function's `trailingSlash: false` option if you configured your astro.config.mjs to `trailingSlash: "never"`.
+
+```//rss.xml.js
+export async function GET(context) {
+  return rss({
++   trailingSlash: false,
+	...
+  });
+}
+-   link: `/posts/${post.slug}/`,
++   link: `/posts/${post.slug}`,
+```
```

Comment:
```suggestion
```ts title="src/pages/rss.xml.js" ins={9}
import rss from '@astrojs/rss';

export function GET(context) {
  const posts = Object.values(postImportResult);
  return rss({
    title: 'Buzz’s Blog',
    description: 'A humble Astronaut’s guide to the stars',
    site: context.site,
    trailingSlash: false,
    items: posts.map((post) => ({
      link: post.url,
      ...post.frontmatter,
    })),
  });
}
```
```